test/command/1592.md

```diff
@@ -47,3 +47,9 @@ pandoc -f native -t markdown
 [hi]{.smallcaps}
 ```
 
+```
+% pandoc -f html -t native
+<bdo dir="ltr">foo</bdo>
+^D
+[Plain [Span ("",[],[("dir","ltr")]) [Str "foo"]]]
```

Comment:
Note that I can't explain why this is wrapped in a `[Plain ...]` block.

Reply:
The readers all return a full `Pandoc` document, so a bare inline expression has to be wrapped in some kind of block. This ends up being `Plain` for the html reader.

Reply:
If I understand correctly I don't need to update the `pBdo` and this output is expected?

Reply:
It's expected, yes. You'd have to modify `readHtml` or `writeNative` themselves not to have it.